packages/react-native-windows-init/src/Cli.ts

```diff
@@ -298,6 +298,24 @@ function installReactNativeWindows(
     );
   }
 
+  const parsedVersion = semver.parse(version);
+  if (parsedVersion) {
+    const newSteps = "Please see https://microsoft.github.io/react-native-windows/docs/getting-started for the latest method for adding RNW to your project.";
+    if (parsedVersion.minor > 75
+      || (parsedVersion.minor === 0
+          && parsedVersion.prerelease.length > 1
+          && parsedVersion.prerelease[0] === 'canary'
+          && typeof parsedVersion.prerelease[1] === 'number'
+          && parsedVersion.prerelease[1] > 843)
```

Comment:
just curious what is "parsedVersion.prerelease[1] > 843" checking?

Reply:
I want to throw a warning for > 0.0.0-canary.843.